src/app/harbor/battles/battles.tsx

```diff
@@ -265,41 +265,91 @@ export default function Matchups({ session }: { session: HsSession }) {
     { retryTimeout }: { retryTimeout: number } = { retryTimeout: 4000 }
   ) => {
     setLoading(true);
-    try {
+    const isTutorial = sessionStorage.getItem("tutorial") === "true";
+
+    if (isTutorial) {
+      setMatchup({
+        project1: {
+          id: "hack-club-site",
+          title: "Hack Club Site",
+          repo_url: "https://github.com/hackclub/site",
+          deploy_url: "https://hackclub.com",
+          screenshot_url:
+            "https://cloud-lezyvcdxr-hack-club-bot.vercel.app/0image.png",
+          readme_url:
+            "https://raw.githubusercontent.com/hackclub/site/refs/heads/main/README.md",
+          credited_hours: 123,
+          total_hours: 123,
+          rating: 0,
+          entrant: [],
+          contest: [],
+          matchups: [],
+          matchups_count: 0,
+          wins: [],
+          wins_adjustments: 0,
+          losses: [],
+          losses_adjustments: 0,
+          autonumber: 0,
+          entrant__slack_id: [],
```

Comment:
Do we need all these fields like wins, wins_adjustments, rating, etc? Are these there just because those fields are there on what we currently get from the API?